packages/components/src/PrimaryToolbar/Actions.js

```diff
@@ -70,6 +70,7 @@ class Actions extends Component {
                                 onSelect && onSelect(...props);
                                 this.toggleOpen(false);
                             } }
+                            ouiaId="ActionsDropdown"
```

Comment:
Here and in other places. No need to specify a type of the component in the id. `Dropdown` component automatically adds `data-ouia-component-type` based on the component name. Thus here `Actions` will be enough.

Reply:
@quarckster  so, for example, for `ConditionalFilter` will be enough to set the same `ouiaId="ConditionalFilter"` for `Dropdown` and `Toggle` components but we will separate them by `data-ouia-component-type`, correct?

Reply:
Right, but ensure that there are no other Dropdowns or Toggle with the same ouiaId on the page.

Reply:
Thanks for the good tip! Fixed